docs/use-mainnet/fund.mdx

```diff
@@ -94,4 +105,60 @@ Not all available tokens will drip directly onto Linea. If they are, you will se
 
 :::
 
-Specific tokens require specific bridges. If you want to bridge from Goerli to Linea, you can find the tokens, contract addresses, and associated bridges [here](./info-contracts.mdx#token-contract-addresses-and-bridges).
\ No newline at end of file
+Specific tokens require specific bridges. If you want to bridge from Goerli to Linea, you can find the tokens, contract addresses, and associated bridges [here](./info-contracts.mdx#token-contract-addresses-and-bridges).
+  </TabItem>
+  
+  <TabItem value="mainnet">
+  Buying tokens directly on Linea is a straightforward process, using MetaMask Portfolio's Buy feature. 
+  
+  :::info
+  ### If you already have tokens on another network, you can [follow the instructions here](./bridges-of-linea/bridging-non-dev.mdx) to bridge them to Linea.
```

Comment:
imo the `h3` is a bit much -- the callout highlights the point sufficiently.

Reply:
fixed